src/find/mod.rs

```diff
@@ -219,6 +219,18 @@ mod tests {
 
     use super::*;
 
+    #[cfg(windows)]
```

Comment:
The proper way to do this is to use path::join and path::to_string.

But that's more verbose, and for *tests* I'm happy to use a simple search'n'replace bodge.